RELEASE.md

```diff
@@ -4,18 +4,24 @@
 
 A release for the bpfman project is comprised of the following major components:
 
-- bpfman binaries
-- Core GRPC API protobuf definitions
-- Kubernetes Custom Resource Definitions (CRDs)
-- Corresponding go pkg in the form of `github.com/bpfman/bpfman` which includes the following:
+- bpfman and bpfman-api(Core GRPC API protobuf definitions) library crates
```

Comment:
```suggestion
- bpfman (Core library) and bpfman-api (Core GRPC API protobuf definitions) library crates
```
Didn't look right with no space between `bpfman-api` and `(Core ...`. Wasn't sure it was done to show it belonged to `bpfman-api` or not, so add a description to `bpfman`.

Reply:
True!